### PR TITLE
[DOCFIX] Update HDFS docs

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@calvinjia PTAL

File: docs/en/ufs/HDFS.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ $ bin/alluxio format
 $ bin/alluxio-start.sh local
 ```
 
-This will start one Alluxio master and one Alluxio worker locally. You can see the master UI at
+You may need to adjust the mount option if your ramdisk is not mounted. Check `bin/alluxio -h` for the different mount options. This will start one Alluxio master and one Alluxio worker locally. You can see the master UI at
```
Comment (apc999) [inline docs/en/ufs/HDFS.md]: "bin/alluxio -h" -> "bin/alluxio" -h is not supported

File: docs/en/ufs/HDFS.md [hunk 2]
```diff
@@ -192,9 +200,18 @@ ALLUXIO_JAVA_OPTS+=" -Djava.security.krb5.realm=<YOUR_KERBEROS_REALM> -Djava.sec
 ### Mount HDFS with Specific Versions
 
 There are multiple ways for a user to mount an HDFS cluster with a specified version as an under storage into Alluxio namespace.
+Before mounting HDFS with a specific version, make sure you have built a client with that specific version of HDFS.
```
Comment (apc999) [inline docs/en/ufs/HDFS.md]: after 2.0, we support 1.2, 2.2 and 2.7 by default, for these versions you don't need to build hdfs.
for other hdfs versions, we need to build it as you indicated here. I think we can perhaps use some different hadoop version to illustrate this, e.g. some cdf versions? also show how to mount this cdf hdfs ufs connector

File: docs/en/ufs/HDFS.md (hunk 1 above)
Comment (calvinjia) [inline docs/en/ufs/HDFS.md]: Could we make this slightly more verbose, mentioning the `SudoMount` option and noting to do that if this is their first time/they have not mounted the ramdisk yet?

File: docs/en/ufs/HDFS.md (hunk 2 above)
Comment (yuzhu) [reply docs/en/ufs/HDFS.md]: if they are building from source, I don't think 2.7 gets built by default. I was using 2.7 and had to go through this step to get it working.  Maybe just tell them to check that the correct jar exists and build if it does not exist.

File: docs/en/ufs/HDFS.md (hunk 1 above)
Comment (yuzhu) [reply docs/en/ufs/HDFS.md]: rewrote so we don't mention this imaginary option..

Comment (yuzhu) [reply docs/en/ufs/HDFS.md]: done

**Comment by yuzhu** [comment]
@calvinjia @apc999 PTAL thanks.

File: docs/en/ufs/HDFS.md [hunk 3]
```diff
@@ -80,6 +80,12 @@ $ bin/alluxio format
 $ bin/alluxio-start.sh local
 ```
 
+If your ramdisk is not mounted, likely because this is the first time you are running Alluxio, you may need to start Alluxio with the `SudoMount` option.
+
+```bash
+$ bin/alluxio-start.sh local SudoMount
+```
```
Comment (calvinjia) [inline docs/en/ufs/HDFS.md]: Include both with and without `SudoMount` so they don't get confused about what to do if they are not running for the first time (ie. they might skip the command altogether).

Comment (yuzhu) [reply docs/en/ufs/HDFS.md]: the command above this shows without `SudoMount` option.

**Pull request merged**